.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "secret_tasks"]
+	path = secret_tasks
+	url = git@github.com:turian/hear2021-secret-tasks.git
```

Comment:
What about people who DON'T have access to this repo?

Do we just tell them not to git submodule update? How do we make sure they don't break?

Can you test this unauthenticated (https maybe) with github? i.e. heareval without auth and without this submodule?